deckhouse-controller/crds/doc-ru-module.yaml

```
@@ -25,6 +25,15 @@ spec:
                 version:
                   description: Версия модуля.
                 requirements:
-                   description: Зависимости модуля.
+                  description: Необходимые требования для включения модуля.
+                  properties:
+                    deckhouse:
+                      description: Требование к версии Deckhouse.
+                    kubernetes:
+                      description: Требование к версии Kubernetes.
+                    bootstrapped:
+                      description: Требование к bootstrap-статусу кластера.
```

```suggestion
                      description: Требование к статусу установки кластера (только для встроенных модулей DKP).
```
@miklezzzz am I right?

---

hard to say, if we mention the `bootstrapped` status somewhere in the documentation, than I'm inclined to agree, otherwise things get complicated :-D

---

Yeah, Here is the [article](https://deckhouse.io/products/kubernetes-platform/documentation/v1/module-development/dependencies/#cluster-installation-status-dependency)